src/views/CardStack/CardStackStyleInterpolator.js

```diff
@@ -53,30 +55,30 @@ function forHorizontal(
   }
 
   const index = scene.index;
-  const inputRange = [index - 1, index, index + 1];
-
   const width = layout.initWidth;
```

Comment:
nit: could you move this constant down to where it is actually being used? There are a couple more places in the code where you create a const and use it much later. Can you please move these assignments closer to where they are put into use? Other than that this looks good to me, so we'll merge. Thanks :)

Reply:
I can not understand your intention, you said the need to modify the place, the original document is so written. Is this modification necessary?

Reply:
hi, what I have in mind is this: imagine you need some object on line 1000 of your code. You only need it in that one place. Where are you gonna create that object? - probably on line 999 because creating it on line 200 will be confusing for the reader. To give an example (read the comments):

```js
  const index = scene.index; // You think: okay, we now have a variable called index
  const interpolate = animatedInterpolate(props); // okay, here we call some function

  if (!interpolate) return { opacity: 0 }; // oh, if the function returned some falsy value, we return early. Why did we create the index variable?

  const opacity = position.interpolate({
    inputRange: ([interpolate.first, index, interpolate.last]: Array<number>),
    outputRange: ([0, 1, 1]: Array<number>),
  });
```
Do you see what I mean? It's just to improve readability, it will not affect the results. Can you do the changes? Thanks.